nilearn/plotting/find_cuts.py

```diff
@@ -209,6 +210,13 @@ def find_cut_slices(img, direction='z', n_cuts=12, spacing='auto'):
 
     data = _smooth_array(data, affine, fwhm='fast')
 
+    if (isinstance(n_cuts, numbers.Number) and n_cuts > 0):
```

Comment:
> I would be in favor of requiring n_cuts to be an integer personally as
> mentioned before.

No. Please don't test on types. People (beginners) don't understand what
the difference between "0" and "0." is. And they shouldn't have to
understand that to use nilearn. Test that it is very close to an integer.